modules/Prospects/metadata/listviewdefs.php

```diff
@@ -54,7 +54,7 @@
         'label' => 'LBL_LIST_TITLE', 
         'link' => false,
         'default' => true),   
-    'EMAIL1' => array(
+    'EMAIL1_LINK' => array(
```

Comment:
In modules such as accounts, the listviewdefs are defined with the field name set to EMAIL1 and the customCode with EMAIL1_LINK. For Example:
```php
'EMAIL1' => 
  array (
    'width' => '15%',
    'label' => 'LBL_EMAIL_ADDRESS',
    'sortable' => false,
    'link' => true,
    'customCode' => '{$EMAIL1_LINK}',
    'default' => true,
  ),
```
What is the impact of changing this fields to EMAIL1? Is it likely to break existing customisations?